db/cyndi_setup_test.sql

```diff
@@ -36,5 +36,5 @@ FROM inventory.hosts_v1_1;
 --
 -- These are the host seeds for use with testing locally and in CI
 --
-INSERT INTO inventory.hosts_v1_1 VALUES ('5e65ac3f-8b44-4f60-9e99-abdafb31740c', '00001', 'MyStringone', '[{"namespace": "insights-client", "key": "env", "value": "prod"}]', '2020-07-21 05:35:53.682554+00', '2020-07-21 05:35:53.682554+00', '2030-07-21 05:35:53.682554+00', '{"sap_system": true}');
+INSERT INTO inventory.hosts_v1_1 VALUES ('5e65ac3f-8b44-4f60-9e99-abdafb31740c', '00001', 'MyStringone', '[{"namespace": "insights-client", "key": "env", "value": "prod"}]', '2020-07-21 05:35:53.682554+00', '2020-07-21 05:35:53.682554+00', '2030-07-21 05:35:53.682554+00', '{"sap_system": true, "operating_system": {"major": 7, "minor": 4, "name": "RHEL"}}');
```

Comment:
In order to test this, I had to add some OS info here. Only one side effect was produced in the `xccdf/test_result_test.rb` below, which was fixed with a stub.